pom.xml

```diff
@@ -203,6 +203,12 @@
         <artifactId>asm</artifactId>
         <version>9.3</version>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>2.0.6</version>
```

Comment:
Where is this used? Adding only to `dependencyManagement` and not `dependencies` seems to have no effect?

Reply:
it's part of dependency section below to tell which version we want to have - in this PR it's only refactoring because we move slf4j-api from provided to compile dependencies